CardanoSharp.Wallet/CardanoSharp.Wallet.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <LangVersion>9.0</LangVersion>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>3.0.0</Version>
+    <Version>3.0.1</Version>
```

Comment:
There seems to be breaking changes to the main SDK surface (i.e. IChangeCreationStrategy, ICoinSelectionService, TransactionOutputExtensions etc.) which should be a major version bump?

Reply:
This PR was really supposed to be out with 3.0.0 but was split for the purposes of not having an 8000 line PR

Reply:
we can bump to `4.0.0` no issue with that

Reply:
Yeah I would suggest `4.0.0` I think there are quite a few consumers using these methods (myself included)

Reply:
👍